src/enclave/rpc_context.h

```diff
@@ -193,7 +172,6 @@ namespace enclave
       const std::string_view& name) = 0;
 
     virtual const std::vector<uint8_t>& get_serialised_request() = 0;
-    virtual std::optional<ccf::SignedReq> get_signed_request() = 0;
```

Comment:
`SignedReq` structs can only be retrieved if we used that signature for auth, and have a signed identity. They're not stored on the request context, so this getter is goner.